app/frontend/components/options_menu.js

```diff
@@ -114,6 +114,17 @@ class OptionsMenu {
       $(this._dictionaryOption).hide();
     }
 
+    this._languageSwitcherOption = this.initConfigOption('localeSwitchOption', async () => {
+      const newLang = this._languageSwitcherOption.value;
+      console.log('Language changed', newLang);
+
+      await i18n.changeLanguage(newLang);
+      $(document).localize();
+
+      await app_controller.tab_controller.loadTabConfiguration();
```

Comment:
I'm not sure what to call to re-render UI with new locale. I got lost in following the loading process while jumping from controller to controller.
`loadTabConfiguration()` seems to be working but gives this error:
```
/home/zhuiks/workspace/ezra-project/app/frontend/controllers/text_controller.js:44 Uncaught (in promise) TypeError: Cannot read property 'resetSearch' of null
    at TextController.prepareForNewText (/home/zhuiks/workspace/ezra-project/app/frontend/controllers/text_controller.js:44)
    at TabController.populateFromMetaTabs (/home/zhuiks/workspace/ezra-project/app/frontend/controllers/tab_controller.js:262)
    at async TabController.loadTabConfiguration (/home/zhuiks/workspace/ezra-project/app/frontend/controllers/tab_controller.js:312)
    at async /home/zhuiks/workspace/ezra-project/app/frontend/components/options_menu.js:124
    at async HTMLElement.<anonymous> (/home/zhuiks/workspace/ezra-project/app/frontend/components/options_menu.js:141)
```
@tobias-klein do you have any suggestions?

Reply:
I don't think we need this! I removed it ... consider my latest commits.